images/wallet-web/templates/default.conf.template

```diff
@@ -93,7 +100,8 @@ server {
             "key-type": "${DIDCOMM_KEY_TYPE}",
             "key-agreement-type": "${DIDCOMM_KEY_AGREEMENT_TYPE}",
             "web-socket-read-limit": ${WEB_SOCKET_READ_LIMIT},
-            "kms-server-url":"${KMS_SERVER_URL}"
+            "kms-server-url":"${KMS_SERVER_URL}",
+            "enableDIDComm":"${ENABLE_DIDCOMM}",
```

Comment:
Can we make `enableDIDComm` as a bool ? We can refer `use-edv-batch` for similar use.